services/background.py

```
@@ -273,6 +280,40 @@ async def _stop_hermes(self) -> None:
+    try:
+        from agent.schedule_store import ScheduleStore
+        from services.warmup import warmup_step
+
+        deadline = asyncio.get_running_loop().time() + _SCHEDULER_HYDRATE_BUDGET_SEC
+        n = await warmup_step(
+            scheduler.attach_persistence_async(ScheduleStore()),
+            "scheduler hydration", deadline, _SCHEDULER_HYDRATE_BUDGET_SEC,
+        )
+        if n is not None:
+            log.info("Scheduler durable persistence attached (%d job(s) rehydrated)", n)
+    except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)